RELEASE/scripts/autoscend/paths/legacy_of_loathing.ash

```diff
@@ -39,10 +39,13 @@ boolean lol_buyReplicas()
 		{
 			buy($coinmaster[Replica Mr. Store], 1, $item[Replica 2002 Mr. Store Catalog]);
 		}
-		if(contains_text(page, "patriotic eagle")) //2023
+		if(!is100FamRun()) //If this isn't a 100% familiar run, go ahead and get another familiar
```

Comment:
I'm not good with making code suggestions in github, but here is pseudo code. Idea is to keep general formatting of initially checking for what is on the page, eagle in this case. Also to be ok to use it if the 100 fam run is the eagle

```
if(contains eagle)
{
         if(!is100FamRun() || (is100FamRun() && get_property("auto_100familiar").to_familiar() == $familiar[patriotic eagle]))
         {
               buy and use
        }
}

Reply:
Note I didn't do any syntax checking, this is just to give ya an idea of what I think will work well

Reply:
My first commit looked very close to your pseudo code, and it had a logic flaw (fixed by moving things around in the second commit. (I have it commented below).

Here's why I approached this the way I did.

- If we're not doing a 100% fam run, we should buy it the first chance we get (that's how the script has been approaching 2023 IoTMs, always buy)
- In a 100% fam run, we won't buy the eagle, ever, so we don't need to even check to see if it's on the page. 
- Since the eagle is from 2023 (in season), it is always available in the store every time we have a Replica Mr. A. 
-The 100% fam logic does not support buying any optional familiars. It assumes the Adventurer has already purchased up to the desired familiar hatchling on turn 0. (reduces error checking)

Specifics on why the initial commit code broke are in the (outdated) comment below.

Reply:
All good points and agree